### PR TITLE
Migrate to resque scheduler

### PR DESCRIPTION
**Pull request by joecannatti**
This allows us to stop depending on the Heroku addon "Heroku Scheduler", which does not guarantee the running of jobs.

### CODE REVIEW

File: app/jobs/monitor_job.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class MonitorJob
+
+  def self.perform(checker)
+    Rake::Task["monitor:#{checker}"].invoke
```
Comment (davetron5000) [inline app/jobs/monitor_job.rb]: Should we inline the rake task and remove it?

File: config/initializers/resque.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  Resque.redis = Redis.new(:host => uri.host, :port => uri.port, :password => uri.password)
+else
+  Resque.redis = Redis.new
+end
+
+require 'resque/failure/redis'
+Resque::Failure::MultipleWithRetrySuppression.classes = [Resque::Failure::Redis]
+Resque::Failure.backend = Resque::Failure::MultipleWithRetrySuppression
+
+Resque::Plugins::Status::Hash.expire_in = 24.hours.to_i
```
Comment (davetron5000) [inline config/initializers/resque.rb]: I don't think we need the resque status plugin do we?

File: app/jobs/monitor_job.rb (hunk 1 above)
Comment (joecannatti) [inline app/jobs/monitor_job.rb]: Yeah, I think so. Will do

File: config/initializers/resque.rb (hunk 2 above)
Comment (joecannatti) [inline config/initializers/resque.rb]: Copied from another app's config. Wasn't sure. Reading more about it now, no I don't think we do.

**Pull request merged**